Formula/bit.rb

```diff
@@ -20,7 +20,6 @@ def install
   end
 
   test do
-    assert_equal "successfully initialized an empty bit scope.\n",
-                 shell_output("#{bin}/bit init --skip-update")
+    system bin/"bit", "init", "--skip-update"
```

Comment:
Hi dunn,
yes the printed message did change, it is now depends on the language the user is working with.
so the message can change according to language.
thats why the only way to assure that bit is working is by running bit init and having it return a zero exit code.